src/widget.ts

```diff
@@ -81,25 +81,24 @@ class MCPTableView extends DOMWidgetView {
   // http://codebeerstartups.com/2012/12/how-to-improve-templates-in-backbone-js-learning-backbone-js/
   tableTemplate =  _.template( '<% for (let elementRow of elementTable)'+
   ' { print("<div class=\'periodic-table-row\'>"); for (let elementName of elementRow)'+
-  ' { if ( (elementName === "") || (elementName == "*" ) ) { %>' +
+  ' { if ( (elementName === "") || (elementName == "*" ) || (elementName == "#" ) ) { %>' +
```

Comment:
I guess `*` is not needed anymore due to the change above from `*` -> `#`?
```suggestion
  ' { if ( (elementName == "") || (elementName == "#" ) ) { %>' +
```

Reply:
I used both "#" and "*" to separate lanthanides and actinides group.